geth/account/importer_test.go

```diff
@@ -0,0 +1,80 @@
+package account
+
+import (
+	"crypto/ecdsa"
+	"fmt"
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/golang/mock/gomock"
+	"github.com/status-im/status-go/extkeys"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestManager_ImportExtendedKey_Success(t *testing.T) {
```

Comment:
Does this name pass a linter?

Reply:
@themue Yes, of course. What does the problem with this name?

Reply:
Ah, just tested here. My linter is fine with underscores in tests but dislikes them in production code. Okay.